src/components/auto-complete/auto-complete.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { SBComponent, SBStateless } from 'stylable-react-component/dist/stylable-react';
+import { root } from 'wix-react-tools';
+import { OptionList, SelectionItem, SelectionList } from '../selection-list/selection-list';
+import style from './auto-complete.st.css';
+
+// Selected item is a string because of selection list's constraints
+// i would love to implement it like the TreeView where the reference to an object
+// was the way to point at the selected item
+
+export type FilterPredicate = (item: string, filterString: string) => boolean;
+
+export interface AutoCompleteListProps {
+    open: boolean;
+    items?: string[];
+    onItemClick?: (item: string) => void;
+}
+
+export const AutoCompleteList: React.SFC<AutoCompleteListProps> = SBStateless(props => {
+    if (props.open) {
+        return (
+            <div data-automation-id="AUTO_COMPLETE_LIST">
+                <SelectionList
+                    className="auto-complete-list"
+                    dataSource={props.items}
+                    onChange={props.onItemClick!}
+                />
+            </div>
+        );
+    }
+    return null;
+}, style);
+
+export interface AutoCompleteProps extends OptionList {
+    value?: string;
+    onChange?: (value: string) => void;
+    style?: any;
```

Comment:
`style` and `className` shouldn't be explicit props. We'll deal with them on a general level.